LICENSE.txt

```diff
@@ -0,0 +1,897 @@
+Mattermost Licensing
+
+SOFTWARE LICENSING
+
+You are licensed to use compiled versions of rtcd produced by Mattermost, Inc. under an MIT LICENSE
+
+-	See MIT-COMPILED-LICENSE.md included in compiled versions for details
```

Comment:
Will we include or generate MIT-COMPILED-LICENSE.md after build ?

Reply:
Good point. This is ported from `mattermost-server` to be consistent. I believe this is what we include in our team edition package. That said, not sure what a MIT applied to compiled software means to be honest but for consistency's sake I think we can include it.

This would be the file:

[MIT-COMPILED-LICENSE.md](https://github.com/mattermost/rtcd/files/8830116/MIT-COMPILED-LICENSE.md)

Reply:
@pfltdv Let me know how you'd like to proceed. Happy to create a separate ticket to address this.